src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
@agutoli so getContext() return the same instance but *without the fields being excluded* or *only including the include fields* ? 
so is it possible to .getContext(context).getContext(context).getContext(context).
Should we work on a contextualEntity, so we just allow *ONE* getContext ? 
With this approach, .getContext() wil return a *contextual entity*.